test/export/test_passes.py

```diff
@@ -87,6 +94,55 @@ def _get_output_names(gm: torch.fx.GraphModule) -> List[str]:
     return [str(arg) for arg in args]
 
 
+class ModelsWithScriptObjectAttr:
+    load_torchbind_test_lib()
+
+    class Simple(torch.nn.Module):
+        def __init__(self):
+            super().__init__()
+            self.attr = torch.classes._TorchScriptTesting._Foo(10, 20)
+
+    class SimpleWithAttrInContainer(torch.nn.Module):
+        def __init__(self):
+            super().__init__()
+            self.attr = torch.classes._TorchScriptTesting._Foo(10, 20)
+            self.pytree_attr2 = [
```

Comment:
currently, _gather_constant_attrs doesn't recursively gather the constants inside pytree container attributes. Need to fix it in the future.

Reply:
Please add a test that the pytree thing doesn't work (and label it with an expectedFailure or something). Maybe also an issue